buildSrc/src/main/kotlin/io/spine/gradle/report/pom/PomGenerator.kt

```diff
@@ -68,8 +68,10 @@ object PomGenerator {
     fun applyTo(project: Project) {
 
         /**
-         * In some cases, the `base` plugin, which is by default is added by e.g. `java`,
-         * is not yet added. `base` plugin defines the `build` task. This generator needs it.
+         * In some cases, the `base` plugin, which by default is added by e.g. `java`,
+         * is not yet added.
+         * The `base` plugin defines the `build` task.
```

Comment:
I see it is not a top level comment, but it looks unusual without empty lines between paragraphs.

```suggestion
         *
         * The `base` plugin defines the `build` task, which is needed by the POM generator.
```